storybook/src/components/Button/Button.stories.tsx

```diff
@@ -19,11 +19,6 @@ const meta = {
     children: {
       table: { disable: false },
```

Comment:
Props for not nuking everything of all of the `argTypes` (pun intended). Can we document this config for people to consider?

Reply:
I'm not sure what you mean. Do you mean documenting that you automatically get your typing as controls, but if you want to show something other than the automatic controls, you have to use `argTypes`?

Reply:
No, sorry I meant that hiding `children` from the Args Table is one thing that may still be necessary to configure, next to declaring HTML attributes.

Reply:
Got it, I've added it to the docs